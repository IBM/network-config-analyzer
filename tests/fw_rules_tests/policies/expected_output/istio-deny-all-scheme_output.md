|query|src_ns|src_pods|dst_ns|dst_pods|connection|
|---|---|---|---|---|---|
|istio-deny-all, config: istio-deny-all||||||
|||ip block: 0.0.0.0/0|[kube-system,vendor-system]|[*]|TCP 1-65536,|
||[default,kube-system,vendor-system]|[*]|[kube-system,vendor-system]|[*]|TCP 1-65536,|


