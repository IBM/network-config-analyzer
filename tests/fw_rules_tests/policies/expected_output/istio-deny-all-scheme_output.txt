For connections of type TCP, final fw rules for query: istio-deny-all, config: istio-deny-all:
src: 0.0.0.0/0 dst_ns: [kube-system,vendor-system] dst_pods: [*] conn: All connections
src_ns: [default,kube-system,vendor-system] src_pods: [*] dst: 0.0.0.0/0 conn: All connections
src_ns: [default,kube-system,vendor-system] src_pods: [*] dst_ns: [kube-system,vendor-system] dst_pods: [*] conn: All connections

For connections of type non-TCP, final fw rules for query: istio-deny-all, config: istio-deny-all:
src: 0.0.0.0/0 dst_ns: [default,kube-system,vendor-system] dst_pods: [*] conn: All connections
src_ns: [default,kube-system,vendor-system] src_pods: [*] dst: 0.0.0.0/0 conn: All connections
src_ns: [default,kube-system,vendor-system] src_pods: [*] dst_ns: [default,kube-system,vendor-system] dst_pods: [*] conn: All connections
